### ISSUE TITLE
Can't select text in search box

### ISSUE THREAD

**Issue by silverwind** [bitwarden/desktop#1357]
### Steps To Reproduce

1. Open the app on macOs
2. Type something into the search box at the top of the window
3. Try to select text in the search box via drag-and-drop, it unexpectedly moves the window instead

### Expected Result

Text to be selectable

### Actual Result

The window is moved by the drag-and-drop action

### Screenshots or Videos

<img width="464" alt="image" src="https://user-images.githubusercontent.com/115237/154444929-5d94fba9-82fc-4adf-8918-c1ec3707b145.png">

### Operating System

macOS

### Operating System Version

12.2

### Installation method

Homebrew

### Build Version

1.31.3

**Comment by kapitainsky** [comment]
Yes it does not work with a mouse but is still possible with keyboard (hold Shift and then press <- or ->)

**Comment by silverwind** [comment]
I think the general expectance is that mouse selection should work in a text box. From what I understand, the mouse events need to be processed by the text box (e.g. not call `.preventDefault()` and not handle the event in a parent node).

**Comment by kapitainsky** [comment]
agree - it should be possible to select text with cursor. This is definitely broken right now.
